src/immer/withImmer.ts

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable import/named */
+import { produce, Draft } from 'immer'
+import { atom, WritableAtom } from 'jotai'
+
+export const withImmer = <Value>(anAtom: WritableAtom<Value, Value>) =>
```

Comment:
In the future, we may try using different types for Update: `WritableAtom<ValueToRead, ValueToWrite>`

Reply:
I found the solution for this and [this](https://github.com/pmndrs/jotai/pull/120#pullrequestreview-505628651), it gave me error at first, but now, I converted them to unknown then to the Update type. [codesandbox](https://codesandbox.io/s/jotai-immer-typescript-7s1fp?file=/src/App.tsx)

Reply:
No, `as unknown` is not nice for this one. Let's just leave it.